pkg/app/app.go

```diff
@@ -121,7 +121,7 @@ func CreateApp() App {
 
 	app.log.Debug("Creating AnsibleBroker")
 	if app.broker, err = broker.NewAnsibleBroker(
-		app.dao, app.log.Logger, app.config.Openshift, app.registry, *app.engine,
+		app.dao, app.log.Logger, app.config.Openshift, app.registry, *app.engine, app.config.LaunchApbOnBind,
```

Comment:
-1 instead of passing in a boolean, we should be looking to pass in a Broker config. That will give us flexibility to add other settings.